Naming.java

```diff
@@ -0,0 +1,31 @@
+package az.alizeynalli.cleancode.naming;
+
+public class Naming {
+
+    public Date d;
+    public List<Testcase> find(User user){};
+    
+    public Account[] AccountList;
+
+    public void deleteUser(User pUser){};
```

Comment:
//No need for prefix p for parameters
    // Good
    public void deleteUser(User user){};